.github/workflows/release_manager.yml

```diff
@@ -0,0 +1,71 @@
+name: Release Manager
+
+on:
+  push:
```

Comment:
This PR keeps trying to release. In fact, it's blocked right now. Can you fix that?
I think we may need another trigger. We probably want to trigger on `master` only. Also, perhaps you need to remove the tags.

Reply:
See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

Let's restrict to the main branch only.

Reply:
I think a `branch:` filter would cause it to trigger on every push to master, which seems undesirable.  The current configuration should only be triggering when a tag is created, which is much less frequent.

Maybe the attempts to release are just Jon testing the release script by adding tags to this branch?

Reply:
That's what the releases are.  The tests are hanging I think because of https://github.com/actions/virtual-environments/issues/3287

Reply:
Upgraded Ubuntu versions in f5fd9232964bf0fedf9f333772ae7b86ff29f24c

Reply:
@bemasc I'm asking to add both the branch and the tag restriction. My understanding is that that's an "AND".
So only if you tag master this will trigger.

Reply:
I think it's an "OR".  See [this discussion](https://github.community/t/run-workflow-on-push-tag-on-specific-branch/17519).

Reply:
Ok, I think that's relatively safe, since only writers can push tags. Please remove the test tags.